### ISSUE TITLE
🐛 Bug Report: Error calling `account.createPhoneToken()`

### ISSUE THREAD

**Issue by swalih-orieal-collabs** [appwrite/appwrite#8823]
### 👟 Reproduction steps

When I attempt to send an SMS using the Appwrite API, I encounter the following error:

Error: AppwriteException

```
const client = new Client();
client.setEndpoint('https://cloud.appwrite.io/v1').setProject('projectID');

const account = new Account(client);

    sendSms = async (phoneNumber) => {
        try {
            const token = await account.createPhoneToken(ID.unique(), phoneNumber);
            return token;
        } catch (error) {
            console.error('Error sending SMS:', error);
            throw new Error(error);
        }
    };
```

here the code i used

### 👍 Expected behavior

it should be a bug

### 👎 Actual Behavior

it actually make me mad its working smoothly earlier. suddenly every thing is collapsed #/

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [x] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [x] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by gepd** [comment]
Please update your title to be more descriptive.
 
In `createPhoneToken`, you must pass the `userId` not `ID.unique()`

**Comment by Megamannen** [comment]
> In `createPhoneToken`, you must pass the `userId` not `ID.unique()`
Not true, when signing in an already existing user phone number it will throw away this ID.
When it doesn't exist it will use the provided ID and create a new user.


You obviously have printed out the error, but you didn't give it to us, please add it.
Have you configured an SMS-provider?

**Comment by HarshMN2345** [comment]
No repro on this one and nothing since 2024-10-21, so there is nothing to go on. Closing. Happy to reopen if you can still hit it and post the steps.
